src/news_app/news_app.js

```diff
@@ -0,0 +1,52 @@
+import { makeFeed } from './create_feed';
+import { getData } from './get_data';
+import { clearFeed } from './clear_feed';
+// import { updateFeed } from './update_feed.js';
+
+import './news_style.css';
+
+// VARIABLES
+const urlFood = `https://api.thenewsapi.com/v1/news/all?api_token=NfHTFOIIekZazvqWjUldhd6FX72AKBkguV1ye0D2&categories=food&language=en`;
```

Comment:
create a variable for a toke and use string interpolation, same with URL and categories